cves/CVE-2017-15706.yml

```diff
@@ -43,7 +47,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
```

Comment:
This needs to be clearer. What part of the system? What would trigger it? What could be accomplished? What did they do to fix it?